src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java

```diff
@@ -67,7 +68,9 @@ public ErrorMessage handleRuntimeException(RuntimeException e, ServletWebRequest
     }
 
     private ErrorMessage logAndGetErrorMessage(ServletWebRequest request, Exception e, Level level) {
-
+        if (e.getClass().equals(UndeclaredThrowableException.class)){
+            e = (Exception) ((UndeclaredThrowableException) e).getUndeclaredThrowable();
```

Comment:
Could you explain who throws such exception?

Reply:
It's 'validate' method in UserRegistrationDtoValidator thru @SneakyThrows annotation.